core/src/event_id.rs

```diff
@@ -37,15 +37,16 @@ impl EventId {
     pub fn builder() -> Builder<Init> {
         Builder { state: Init }
     }
+
     /// EventId.new builds a Vec<u8> with the event id data.
     pub fn new(
-        network: &Network,
-        sort_key: &str,
-        sort_value: &str,
-        controller: &str,
-        init: &Cid,
-        event_height: u64,
-        event_cid: &Cid,
+        network: &Network, // e.g. Network::Mainnet
+        sort_key: &str,    // e.g. "model"
+        sort_value: &str,  // e.g. "kh4q0ozorrgaq2mezktnrmdwleo1d" // cspell:disable-line
+        controller: &str,  // e.g. "did:key:z6MkgSV3tAuw7gUWqKCUY7ae6uWNxqYgdwPhUJbJhF9EFXm9"
+        init: &Cid, // e.g. Cid::from_str("bagcqceraplay4erv6l32qrki522uhiz7rf46xccwniw7ypmvs3cvu2b3oulq") // cspell:disable-line
+        event_height: u64, // e.g. 1
+        event_cid: &Cid, // e.g. Cid::from_str("bafyreihu557meceujusxajkaro3epfe6nnzjgbjaxsapgtml7ox5ezb5qy") // cspell:disable-line
```

Comment:
This is a nice example. I way to make this show up in the docs is to show an example call in the doc comment.

Something like this:

```rust
///  EventId.new builds a Vec<u8> with the event id data.
/// ## Example
/// ```
/// let event = Event::new(
///     Network::Mainnet,
///     "model",
///     "kh4q0ozorrgaq2mezktnrmdwleo1d",
///     "did:key:z6MkgSV3tAuw7gUWqKCUY7ae6uWNxqYgdwPhUJbJhF9EFXm9",
///     Cid::from_str("bagcqceraplay4erv6l32qrki522uhiz7rf46xccwniw7ypmvs3cvu2b3oulq"),
///     1,
///     Cid::from_str("bafyreihu557meceujusxajkaro3epfe6nnzjgbjaxsapgtml7ox5ezb5qy"),
/// );
///
/// ```
 pub fn new() -> EventId {
 // ...
}
```


You might need to add some `use` statements to make is actually compile and run. But then we can keep the example in sync with the code this way.